faucet/authenticator/test/controllers/index_spec.js

```diff
@@ -34,10 +35,10 @@ describe('twitter controller', () => {
 			generateAuthLinkStub.returns(Promise.resolve(twitterAuthResponse));
 
 			// Act:
-			const result = await twitter.requestToken();
+			const result = await twitter.requestToken(frontendCallbackUrl);
```

Comment:
Should there be a test when the callback is missing from the param?

Reply:
I had another test in `throws error when request token fail`
do you think that is enough because as long as the callback is not valid (with or without params) I will throw the error.